src/capidle/index.html

```diff
@@ -19,9 +19,51 @@
   </head>
   <body>
     <core-navbar>Capidle</core-navbar>
+
+    <!-- Modal window with instructions on how to play -->
+    <core-modal>
+      <div class="modal-header">HOW TO PLAY CAPIDLE</div>
+      <p class="modal-text">
+        The aim is to figure out which city (or town) is shown on the map.
+        <br />
+
+        If your guess is wrong, you will be given a hint:
+      </p>
+      <div class="modal-text">
+        <div class="guess-rows" style="max-width: 300px">
+          <div class="guess">
+            <span>Berlin</span>
+            <span>1,200km</span>
+            <span>↙️</span>
+          </div>
+        </div>
+      </div>
+      <p class="modal-text">
+        The hint tells you the how far away you are from the correct location,
```

Comment:
Maybe you could add a line break here to make the dialogue a bit less wide

Reply:
this was a bit tricky so I fiddled with the css in 3c92164b